New Entry: General Font Awesome (#849)
Codecademy/docs: content/general/concepts/font-awesome/terms/fa-fw/fa-fw.md

---
Title: 'fa-fw'

> Description: 'Used to set an icon to be fixed width.'

Reviewer: ```suggestion
Description: 'Sets an icon to a fixed width.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'World Wide Web'
  - 'Fonts'
CatalogContent:
  - 'learn-css'
  - 'paths/front-end-engineer-career-path'
---

> The **`fa-fw`** class is used to set an icon to be fixed width. This is useful when verically aligning icons in a list.

Reviewer: ```suggestion
The **`fa-fw`** class is used to set an icon to a fixed width. This is useful when vertically aligning icons in a list.
```
